cli/crates/backend/src/api/deploy.rs

```diff
@@ -57,8 +57,10 @@ pub async fn deploy() -> Result<(), ApiError> {
             .map_err(ApiError::AppendToArchive)?;
     }
 
-    let walker = WalkDir::new(&project.path).into_iter();
-    for entry in walker.filter_entry(|entry| entry_not_in_blacklist(entry, &project.path)) {
+    println!("Project path: {}", project.path.to_string_lossy());
```

Comment:
@jakubadamw This should probably be removed

Reply:
Yes, sorry…